demos/mqtt/mqtt_demo_mutual_auth/mqtt_demo_mutual_auth.c

```diff
@@ -553,7 +558,10 @@ static uint32_t generateRandomNumber()
 }
 
 /*-----------------------------------------------------------*/
-static int connectToServerWithBackoffRetries( NetworkContext_t * pNetworkContext )
+static int connectToServerWithBackoffRetries( NetworkContext_t * pNetworkContext,
+                                              MQTTContext_t * pMqttContext,
+                                              bool * pClientSessionPresent,
+                                              bool * pBrokerSessionPresent )
 {
     int returnStatus = EXIT_SUCCESS;
```

Comment:
The loop will only work if this is initialized to failure
```suggestion
    int returnStatus = EXIT_FAILURE;
```